tests/flytekit/unit/core/test_workflows.py

```diff
@@ -1,15 +1,24 @@
 import typing
 from collections import OrderedDict
 
+import pandas as pd
 import pytest
+from pandas._testing import assert_frame_equal
```

Comment:
This is more of a nit, but can we import that function from `pandas.testing` instead? It's present in the [docs](https://pandas.pydata.org/docs/reference/api/pandas.testing.assert_frame_equal.html#pandas-testing-assert-frame-equal).